### ISSUE TITLE
For IOS I received this message "Unable to determine Swift version for the following pods"

### ISSUE THREAD

**Issue by EdinsonNM** [authpass/biometric_storage#3]
[!] Unable to determine Swift version for the following pods:

- `biometric_storage` does not specify a Swift version and none of the targets (`Runner`) integrating it have the `SWIFT_VERSION` attribute set. Please contact the author or set the `SWIFT_VERSION` attribute in at least one of the targets that integrate this pod.

**Comment by hpoul** [comment]
I have now added a swift version to the podspec.. maybe you could try the current master? in your pubspec for example:

```
    flutter_async_utils:
    git:
      url: https://github.com/authpass/biometric_storage.git
      ref: 788b13e4939712845bd40cc378139ccf6d716a9d
```

**Comment by EdinsonNM** [comment]
Thanks @hpoul . Now this works fine for me. I owe you one :)

**Comment by hpoul** [comment]
Great, thanks for testing. I've just published it to pub as 0.3.3.
